SK Post PED

PohSayKeong opened the pull request.

PohSayKeong committed:
Move format of the command before the screenshot

This is to provide more clarity on the command before demoing it with the screenshot.

PohSayKeong committed:
Remove from and to field from add and edit

The from and to fields in add and edit is causing a lot of confusion for users.

These fields are now removed from add and edit commands. User should use editft to modify the freetime.

PohSayKeong committed:
Remove the circle from class diagram

This is to follow the convention for class diagrams for this course.

lshaoqin commented on docs/UserGuide.md:
```diff
@@ -89,16 +89,20 @@ CS2103T Laboratory 10:00-12:00
 ### Adding a default course : `teach`
 
-Adds a default course to the address book.
+The default course is usually the course that you are currently teaching.
```
The language here is not consistent with the other commands

PohSayKeong replied:
resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)

lshaoqin commented on src/main/java/seedu/address/logic/commands/EditCommand.java:
```diff
@@ -183,9 +177,9 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
         /**
          * Returns true if at least one field is edited.
```
Need to change this

PohSayKeong replied:
Resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)

lshaoqin reviewed (changes requested):
Nice job, I suggested two small changes.

PohSayKeong committed:
Add testcases for CourseCommand

Testcases for CourseCommand were missing initially.

This commit will add test cases for CourseCommand and add new cases for various classes.

PohSayKeong committed:
Update description of teach command

This update matches the description of teach command with the rest of the UG.

lshaoqin committed:
Standardise edit command prompt with UG

lshaoqin reviewed (approved):
LGTM
